cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -135,9 +135,10 @@ WholeStageResultIterator::WholeStageResultIterator(
     const auto& format = scanInfo->format;
     const auto& partitionColumns = scanInfo->partitionColumns;
     const auto& metadataColumns = scanInfo->metadataColumns;
+#ifdef GLUTEN_ENABLE_GPU
     // Under the pre-condition that all the split infos has same partition column and format.
```

Comment:
It is used in this line https://github.com/apache/incubator-gluten/pull/11067/files#diff-013353c1aa2ebe3f4c5c014b936dd52b2f91bef0a51421c1a901a57cac361f17L173